interface/repository/pokemon_repository.go

```diff
@@ -3,30 +3,93 @@ package repository
 import (
 	"os"
 
+	resty "github.com/go-resty/resty/v2"
 	"github.com/gocarina/gocsv"
 	"github.com/luispaulin/api-challenge/domain/model"
 )
 
 type pokemonRepository struct {
-	file *os.File
+	file   *os.File
+	client *resty.Client
 }
 
 // PokemonRepository for interface
 type PokemonRepository interface {
 	FindAll(pokemons []*model.Pokemon) ([]*model.Pokemon, error)
+	Sync() (string, int, error)
 }
 
 // NewPokemonRepository for interface
-func NewPokemonRepository(db *os.File) PokemonRepository {
-	return &pokemonRepository{db}
+func NewPokemonRepository(db *os.File, client *resty.Client) PokemonRepository {
+	return &pokemonRepository{db, client}
 }
 
 func (pr *pokemonRepository) FindAll(pokemons []*model.Pokemon) ([]*model.Pokemon, error) {
 
+	// Set reader t file's beginning
```

Comment:
Remove the empty line above

Reply:
`reader t file's` ?

Reply:
Oh, I tried to say: reader at file's, changed.